COOKBOOK.md

```diff
@@ -0,0 +1,81 @@
+
+### Contains list of additional resources
+
+After the procedure section, include a list of additional resources that the consumer might need to be aware of. This could include links to other tools that are expected to be present (e.g. Node.JS when using the @dojo/cli tool) or tutorials and reference guides that expand on one or more concepts that are mentioned in the recipe.
+
+### No boiler plate steps
+
+Recipes should be as targeted and focused as possible. Content that discusses mundane work such as importing modules or installing tools should be left out or presented as tersely as possible. For example, if a new widget is being created, it makes sense to include a step directing the consumer to import the `WidgetBase` module with the complete path since its location might not be intuitive. However, if they need to import a module that the consumer has created, a simple directive to import that module is sufficient.
+
+### Proper casing for titles
+
+All titles should be written in sentence case. Only the first letter and proper nouns should be capitalized.
+
+### Proper voice and tone use
+
+Recipes should focus on being clear and concise. They should adopt a very declarative tone with a minimum of discussion. Avoid decorating the recipe with adjectives and long paragraphs. Instead, short, imperative statements (e.g. "Run {command} in the terminal.") should be used to communicate what the consumer needs to do with as few words as possible.
+
+Note: this tone will probably lead to recipes that feel cold or distant. This is acceptable since the goal is to solve a problem quickly, not teach about the underlying concepts in a manner that the consumer can easily retain.
+
+### No unnecessary contractions or slang
+
+Contractions and slang expression (expressions that are specific to a culture or region) can be confusing to learners that come from other cultures. Since Dojo 2 has been developed to serve a global audience, contractions and slang should be avoided whenever possible. There are, however, some contractions that are acceptable because the contracted words are almost never used separately ("let's" versus "let us"). These uses of contractions are acceptable.
+
+### No pronouns or, if used, have an obvious meaning to non-native English speakers
+
+Many potential users of Dojo 2 are not native speakers of the English language. A common point of confusion with non-native English speakers is determining what noun a pronoun (he, she, it, etc.) is referring. In order to make Dojo 2's tutorials as clear as possible, pronouns should be avoided unless it is extremely obvious what noun is being referred to.
```

Comment:
pronouns should be avoided unless it is extremely obvious what noun is being referred to. -> pronouns should be avoided unless the referenced noun is unambiguous and obvious. (might also want to add a point about not ending a sentence in a preposition)